test/index.html

```diff
@@ -23,7 +23,7 @@ <h2>Scratch Page</h2>
 </ul>
 
 <h2>Manual Tests</h2>
-<a href="manual">Here</a>
+<a href="manual/">Here</a>
```

Comment:
Why is this necessary?

Reply:
The link to manual tests was broken on the `/test` page without trailing slash

I now noticed this is a problem within more sub-directories, e.g all the history subdirectories under `manual` shuld probably have trailing slash as well.